### PR TITLE
Minor Facelift for Landing Page (Dark Mode)

### PR DESCRIPTION
**Pull request by KarateBrot**
Fixes the logo and background pattern for dark mode. The pattern is now more subtle, comparable to the pattern in light mode.

# Before
grafik

# After
grafik

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by TheIsotopes — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
